home/modules/ROOT/pages/index.adoc

```diff
@@ -1,93 +1,256 @@
+WHERE route.sourceairport = "BOS"
+AND route.destinationairport = "SFO"
+AND airline.callsign = "JETBLUE";
+----
 
 ++++
 </div>
 ++++
 
+[.column]
+====== {empty}
+
 == Get Started
 ++++
-<div class="card-row two-column-row">
+<div class="card-row three-column-row">
 ++++
 
 [.column]
 ====== {empty}
-.Couchbase Cloud
+.Couchbase Cloud (DBaaS)
 
 [.content]
-A fully-managed database as a service designed to take the complexity out of deploying, managing, scaling, and securing Couchbase in the public cloud. With just a few clicks, you can store, query, and analyze any amount of data, all on the cloud service provider of your choice.
+Take the complexity out of deploying, managing, scaling, and securing Couchbase in the public cloud. Store, query, and analyze any amount of data, on a cloud service provider of your choice, all in a few clicks. Explore Couchbase Cloud, our fully-managed database as a service offering.
+
+xref:cloud::index.adoc[Couchbase Cloud]
+
+[.column]
+====== {empty}
+.Couchbase Server
 
-xref:cloud::index.adoc[Get Started with Couchbase Cloud]
+[.content]
+Explore Couchbase Server, a modern, distributed document database with all the desired capabilities of a relational database and more. It exposes a scale-out, key-value store with managed cache for sub-millisecond data operations, purpose-built indexers for efficient queries, and a powerful query engine for executing SQL-like queries.
 
+xref:home:server.adoc[Couchbase Server]
 
 [.column]
 ====== {empty}
-.Couchbase Autonomous Operator (Kubernetes)
+.Couchbase Lite
 
 [.content]
-Native integration of Couchbase Server with open source Kubernetes and Red Hat OpenShift to automate the management of common Couchbase tasks such as the configuration, creation, scaling, and recovery of Couchbase clusters.
+Bring the power of NoSQL to the edge with Sync Gateway and Couchbase Lite. Combined with Couchbase Server, it provides fast, efficient bidirectional synchronization of data between the edge and the cloud. Enabling you to deploy your offline-first mobile and embedded applications with greater agility on premises or in any cloud.
 
-xref:operator::overview.adoc[Get Started with Autonomous Operator]
+xref:couchbase-lite::introduction.adoc[Couchbase Lite]
+
+++++
+</div>
+++++
 
 [.column]
 ====== {empty}
-.Couchbase Server
+
+== Developer Tools
+++++
+<div class="card-row three-column-row">
+++++
+
+[.column]
+====== {empty}
+.SDK and Connectors
 
 [.content]
-An open source, distributed, JSON document database. It exposes a scale-out, key-value store with managed cache for sub-millisecond data operations, purpose-built indexers for efficient queries, and a powerful query engine for executing SQL-like queries.
+Couchbase SDKs allow applications to access a Couchbase cluster and the big data Connectors enable data exchange with other platforms.
 
-xref:server:getting-started:start-here.adoc[Get Started: Couchbase Server Enterprise]
+xref:sdk.adoc[Developer Docs]
```

Comment:
Should be * xref:home:ROOT:sdk.adoc[Developer Docs]]
?

Reply:
Since sdk.adoc is in the same directory, we don't need to specify the full path.

Reply:
updated